src/js/components/Tab/Tab.js

```diff
@@ -128,8 +130,17 @@ const Tab = forwardRef(
         onClick={onClickTab}
         onMouseOver={onMouseOverTab}
         onMouseOut={onMouseOutTab}
-        onFocus={onMouseOver}
-        onBlur={onMouseOut}
+        onFocus={() => {
+          // ensure focus outline is not covered by hover styling
+          // of adjacent tabs
+          setZIndex(1);
+          if (onMouseOver) onMouseOver();
+        }}
+        onBlur={() => {
+          setZIndex();
+          if (onMouseOut) onMouseOut();
+        }}
+        style={zIndex && { zIndex }}
```

Comment:
would it make more sense to have the logic around 'focus'  rather than 'zindex'? so something like `style={ focus && {Index: 1 }}`

Reply:
I like that mindset. makes line 143 more readable. I'll update : )

Reply:
Done. : )